L1Trigger/L1TMuonEndCap/src/SectorProcessorLUT.cc

```diff
@@ -27,6 +27,8 @@ void SectorProcessorLUT::read(bool pc_lut_data, int pc_lut_version) {
     coord_lut_dir = "ph_lut_v3_data";  // Update in September 2017 from ReReco alignment, data only
   else if (pc_lut_version == 3 && pc_lut_data)
     coord_lut_dir = "ph_lut_Run3_2022_data";  // Update in October 2022 from Run 3 2022 alignment, data only
+  else if (pc_lut_version == 4 && pc_lut_data)
+    coord_lut_dir = "ph_lut_Run3_2023_data";  // Update in June 2023 from Run 3 2023 alignment, data only
   else if (pc_lut_version >= 2)
```

Comment:
It doesn't change anything, but it would look healtier as such:
```suggestion
  else if (pc_lut_version >= 5)
```

Reply:
This if statement should catcht `pc_lut_version >= 2` when `pc_lut_data` is false. Your suggested version doesn't cover all the cases, if I'm not mistaken. I can clean up the statements next week to simplify them if you prefer that.